packages/snaps-controllers/jest.config.js

```diff
@@ -6,7 +6,7 @@ delete baseConfig.coverageThreshold;
 
 module.exports = deepmerge(baseConfig, {
   coverageDirectory: './coverage/jest',
-  testTimeout: 5000,
+  testTimeout: 10000,
```

Comment:
Why is this required now?

Reply:
I'm not sure. It was consistently failing in CI with 5000ms.